gdk4-x11/src/x11_display.rs

```diff
@@ -0,0 +1,16 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::X11Display;
```

Comment:
Needs to be added to lib.rs

Reply:
That explains why it always was compiling no matter what I did!